Exiled.API/Features/Pickups/Scp244Pickup.cs

```diff
@@ -0,0 +1,106 @@
+// -----------------------------------------------------------------------
+// <copyright file="Scp244Pickup.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features.Pickups
+{
+    using Exiled.API.Features.DamageHandlers;
+    using Exiled.API.Features.Items;
+
+    using InventorySystem.Items.Usables.Scp244;
+
+    using UnityEngine;
+
+    /// <summary>
+    /// A wrapper class for SCP-244.
+    /// </summary>
+    public class Scp244Pickup : Pickup
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Scp244Pickup"/> class.
+        /// </summary>
+        /// <param name="pickupBase">The base <see cref="Scp244DeployablePickup"/> class.</param>
+        public Scp244Pickup(Scp244DeployablePickup pickupBase)
+            : base(pickupBase)
+        {
+            Base = pickupBase;
+        }
+
+        /// <summary>
+        /// Gets the <see cref="Scp244DeployablePickup"/> that this class is encapsulating.
+        /// </summary>
+        public new Scp244DeployablePickup Base { get; }
+
+        /// <summary>
+        /// Gets the speed of <see cref="Scp244Pickup"/>'s too grow.
+        /// </summary>
+        public float GrowSpeed => Base.GrowSpeed;
+
+        /// <summary>
+        /// Gets the time for the sphere to finish their expansion.
+        /// </summary>
+        public float TimeToGrow => Base.TimeToGrow;
+
+        /// <summary>
+        /// Gets the current size effect of the Scp244's Hypothermia.
```

Comment:
it's just a `Ctrl + C Ctrl + V` docs lower

Reply:
It's not in lower it's have percent

Reply:
Oh, i'm blind, okay